db/db_impl.h

```diff
@@ -1345,6 +1449,57 @@ class DBImpl : public DB {
 
   void WaitForBackgroundWork();
 
+  // No copying allowed
+  DBImpl(const DBImpl&);
+  void operator=(const DBImpl&);
+
+  // Background threads call this function, which is just a wrapper around
+  // the InstallSuperVersion() function. Background threads carry
+  // sv_context which can have new_superversion already
+  // allocated.
+  // All ColumnFamily state changes go through this function. Here we analyze
+  // the new state and we schedule background work if we detect that the new
+  // state needs flush or compaction.
+  void InstallSuperVersionAndScheduleWork(
```

Comment:
do you want to sort the methods alphabetically?

Reply:
nah, I don't think sorting works. It would be better to cluster related methods together instead of alpha sorting.